project/Dependencies.scala

```diff
@@ -9,8 +9,12 @@ object Dependencies {
   val AkkaVersion = System.getProperty("override.akka.version", "2.6.3")
   val AkkaVersionInDocs = System.getProperty("override.akka.version", "2.6")
   val CassandraVersionInDocs = "4.0"
-  val DriverVersion = "4.3.0"
-  val DriverVersionInDocs = "4.3"
+  val DriverVersion = "4.5.0"
+  val DriverVersionInDocs = DriverVersion.take(3)
+
+  // Performance dropped by ~40% when the driver upgraded to latest netty version
+  // override for now
+  val OverrideNettyVersion = "4.1.39.Final"
```

Comment:
It's a bit scary to force the dependency to an older version than what the driver was built with. Could result in runtime errors if Netty isn't forwards binary compatible. Anyway, this is hopefully very temporary and our tests should give us some confidence.